src/main/java/ip/response/Response.java

```diff
@@ -0,0 +1,39 @@
+package ip.response;
+
+public class Response {
+    // Horizontal line
+    public static void horizontalLine() {
```

Comment:
function name greeting should be imperative such as "drawHorizontalline" instead of "Horizontalline"

Reply:
Have addressed this. Thanks for the feedback!